### PR TITLE
created logger

### PR DESCRIPTION
**Pull request by kovpaka**

### CODE REVIEW

File: Logger/Logger.cs [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+        public Logger(){}
+
+        public Logger(LoggerHandler loggerHandler, LogLevel highestLogLevel = LogLevel.Error)
+        {
+            Log += loggerHandler;
+            this.highestLogLevel = highestLogLevel;
+        }
+
+        private static string LogMessageBuilder(LogLevel level, string message)
```
Comment (siarheikarabko) [inline Logger/Logger.cs:26]: According to the name, it looks more like a separate class (Builder). Methods usually are actions that are described using verbs.

File: Logger/Logger.cs [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        private readonly LogLevel highestLogLevel;
+
+        public enum LogLevel
+        {
+            Info,
+            Warning,
+            Error
+        }
+
+        public Logger(){}
```
Comment (siarheikarabko) [inline Logger/Logger.cs:18]: What if the logger is instantiated using this constructor?

File: Logger/Program.cs [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            Logger.LoggerHandler lh = PrintLogToFile;
+            lh += PrintLogToConsole;
+            var logger2 = new Logger(lh);
+            logger2.Error("Something went wrong");
+            logger2.Error(new NullReferenceException("Obj is null"));
+
+            Console.ReadKey();
+        }
+
+        public static void PrintLogToConsole(string message)
```
Comment (siarheikarabko) [inline Logger/Program.cs:24]: Let's discuss the solution architecture and logger functionality.

File: Logger/Logger.csproj [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{03B88EE0-0A79-4782-AA63-2D9972EEA179}</ProjectGuid>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>Logger</RootNamespace>
+    <AssemblyName>Logger</AssemblyName>
+    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
```
Comment (siarheikarabko) [inline Logger/Logger.csproj:11]: .Net Framework desires to have a retirement rest, let's leave it and try using .net5/NetStandard :)

File: Logger/Properties/AssemblyInfo.cs [hunk 5]
```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// Общие сведения об этой сборке предоставляются следующим набором
```
Comment (siarheikarabko) [inline Logger/Properties/AssemblyInfo.cs:5]: VS in Russian? Your international colleagues won't be happy with such comments :)

File: Logger/Logger.cs (hunk 2 above)
Comment (kovpaka) [reply Logger/Logger.cs:18]: It's ok because then all logs prints to the console (according to the third item in the task)

**Pull request closed without merging**